spk/git/Makefile

```
@@ -1,6 +1,6 @@
 SPK_NAME = git
-SPK_VERS = 2.37.0
-SPK_REV = 23
+SPK_VERS = 2.37.1
+SPK_REV = 0
```

This must never be decremented.
```suggestion
SPK_REV = 24
```